src/DynamoCoreWpf/ViewModels/Core/WorkspaceViewModel.cs

```diff
@@ -1054,12 +1054,31 @@ private void DoGraphAutoLayout(object o)
         {
             if (Model.Nodes.Count() == 0)
                 return;
+            
+            var selection = DynamoSelection.Instance.Selection;
+
+            bool isInternalLayout = selection.Count > 0 &&
```

Comment:
Does this mean group layout? If so you could perhaps rename it to `isGroupLayout`?